Update supported Kubernetes version (#237)
scalar-labs/helm-charts: docs/how-to-deploy-scalar-products.md

* [scalar-labs/scalar-kubernetes](https://github.com/scalar-labs/scalar-kubernetes/blob/master/README.md)

> You must prepare a Kubernetes with supported versions. See [Supported Kubernetes versions](https://github.com/scalar-labs/helm-charts#supported-kubernetes-versions) for the versions that Scalar Helm Chart supports.

Reviewer: ```suggestion
You must prepare a supported version of Kubernetes. For versions that Scalar Helm Charts supports, see [Supported Kubernetes versions](https://github.com/scalar-labs/helm-charts#supported-kubernetes-versions).
```

Author: Looks good! Thank you for your suggestion!
